src/sentry/models/projectcodeowners.py

```diff
@@ -106,7 +106,7 @@ def validate_codeowners_associations(self, codeowners, project):
                 if project in projects:
                     users_dict[external_actor.external_name] = user.email
                 else:
-                    users_without_access.append(f"{user.username}")
+                    users_without_access.append(f"{user.get_display_name()}")
```

Comment:
```python
def get_display_name(self):
      return self.name or self.email or self.username
```
This might return the user's name is that OK?

Reply:
@mgaeta yea it should be fine bc it'll only return if there's no email or name.